Start time: 06:55
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 300
collection name: collection_spacev10m_sel_1_500_random_100
2025-04-09 21:56:01,893 [ERROR][_create_connection]: Failed to create new connection using: 0af2976f987242169ed20a173d0f1994 (milvus_client.py:923)
Traceback (most recent call last):
  File "utils/pymilvus_ivfpq.py", line 67, in <module>
    client = MilvusClient(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 67, in __init__
    self._using = self._create_connection(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 924, in _create_connection
    raise ex from ex
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 921, in _create_connection
    connections.connect(using, user, password, db_name, token, uri=uri, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 461, in connect
    connect_milvus(**kwargs, user=user, password=password, token=token, db_name=db_name)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 411, in connect_milvus
    gh._wait_for_channel_ready(timeout=timeout)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 155, in _wait_for_channel_ready
    raise MilvusException(
pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530, illegal connection params or server unavailable)>
Command exited with non-zero status 1
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 300 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 2.07
	System time (seconds): 0.10
	Percent of CPU this job got: 20%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:10.68
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 78848
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 25
	Minor (reclaiming a frame) page faults: 22169
	Voluntary context switches: 1982
	Involuntary context switches: 557
	Swaps: 0
	File system inputs: 8840
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
Start time: 06:56
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 300
collection name: collection_spacev10m_sel_1_500_random_100
collection  collection_spacev10m_sel_1_500_random_100  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 100
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 10
ids shape: 10000 10
gt shape: (10000, 10)
milvus get  92850  postive res from  100000  results, recall@ 10 : 0.9285
recall dist(10\% per bucket):  [0.000e+00 1.000e-04 2.000e-04 7.000e-04 1.900e-03 5.900e-03 1.420e-02
 3.520e-02 9.700e-02 3.103e-01 5.345e-01]
10000  queries cost: 118.94772434234619 , qps: 84.07054489936074
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 300 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 21.78
	System time (seconds): 2.89
	Percent of CPU this job got: 16%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 2:26.37
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 91924
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 34
	Minor (reclaiming a frame) page faults: 59934
	Voluntary context switches: 25343
	Involuntary context switches: 514974
	Swaps: 0
	File system inputs: 8096
	File system outputs: 24
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
